src/main/groovy/eu/plumbr/integration/artifactory/ArtifactoryClient.groovy

```diff
@@ -0,0 +1,6 @@
+package eu.plumbr.integration.artifactory
+
+interface ArtifactoryClient {
+
+  List<String> releasedIssues(String buildName, String buildNumber)
```

Comment:
I'd usually agree with you, but here because `released` is a verb, it's a bit confusing.